module/importer/src/Domain/Command/Attribute/ImportTextAttributeCommand.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Domain\Command\Attribute;
+
+class ImportTextAttributeCommand extends AbstractImportAttributeCommand
```

Comment:
Why every Command is "that same" ?
In that case one command its be enough and on handler  side use a resolver to create or update attribute.
That solution is right now.